django-backend/fecfiler/committee_accounts/views.py

```diff
@@ -301,3 +301,33 @@ def update(self, request, *args, **kwargs):
         )
 
         return super().update(request, *args, **kwargs)
+
+    def list(self, request, *args, **kwargs):
+        you_first = request.query_params.get("you_first")
+        logger.info(you_first)
```

Comment:
Was this a debugging aid, or do we want to log it long-term?  I'm fine either way, though.